provider.tf

```diff
@@ -12,6 +12,6 @@ terraform {
 provider "ibm" {
   region           = var.region
   ibmcloud_api_key = var.ibmcloud_api_key
-  generation       = 2
+  # generation       = 2
```

Comment:
if we don't need this remove it. But if we still need to specify a generation then we should keep it and uncomment it.